index.html

```diff
@@ -2905,6 +2915,152 @@ <h3>Evidence</h3>
 
       </section>
 
+      <section>
+        <h3>Confirmation Method</h3>
+
+        <p>
+Confirmation Method can be included by an <a>issuer</a> in a <a>verifiable
+credential</a> to declare that the <a>subject</a> controls one or more
+particular confirmation methods and to bind the <a>claims</a> about the
+<a>subject</a> to one or more of these confirmation methods. In this way, an
+<a>issuer</a> explicitly enables a <a>verifier</a> to validate that the
+<a>holder</a> presenting the <a>verifiable credential</a> has proven control of
+one or more of these confirmation methods when the <a>claims</a> bound to the
+confirmation method are presented.
+        </p>
+
+        <p class="note">
+A <a>verifier</a> can decide to accept <a>claims</a> in a <a>verifiable
+credential</a> without validating the confirmation method or to use a different
+mechanism to validate the <a>holder</a> is bound to the presented <a>claims</a>
+if required without impacting liability if not specified by other means such
+as a <code>termsOfUse</code> policy.
```

Comment:
I'm not sure we should say anything about `liability`, as that's a legal term, and we're not in the business of legalese. But if we're going to, I think the language below is better than that above.
```suggestion
A <a>verifier</a> can decide to accept <a>claims</a> in a <a>verifiable
credential</a> without validating the confirmation method, or to use a different
mechanism to validate whether the <a>holder</a> is bound to the presented <a>claims</a>,
if required. Such decision may prevent impact on the <a>verifier</a>'s
liability or lack thereof if not specified by other means such
as a <code>termsOfUse</code> policy.
```

Reply:
yes, we should probably not touch upon liability. simply sayingrRequirements around the checks the verifier has to perform will be defined elsewhere should be sufficient. something like this:

```suggestion
A <a>verifier</a> can decide to accept <a>claims</a> in a <a>verifiable
credential</a> without validating the confirmation method or to use a different
mechanism to validate the <a>holder</a> is bound to the presented <a>claims</a>.
Requirements around the checks the verifier has to perform will be dictated by 
the trust frameworks or any other means such as a <code>termsOfUse</code> policy.
```